app/assets/stylesheets/_notes.scss

```diff
@@ -71,6 +72,39 @@
       border-color: transparent;
       width: 100%;
       position: relative;
+      height: 28px;
+    }
+
+    .search-options {
+      margin-top: 10px;
+
+      align-self: center;
+      display: flex;
+      flex-direction: row;
+      align-items: baseline;
+      gap: 0.25rem;
+      font-size: var(--sn-stylekit-font-size-p);
+      white-space: nowrap;
+
+      .bubble {
```

Comment:
Great! I've changed to utility classes on https://github.com/standardnotes/web/pull/908/commits/fdecd9c410c74534e8fbc2d0dc959c685224ba40. 

I knew TailwindCSS, bus wasn't familiar with it. Thank you for sharing, really helped! Is SN planning on moving on to it? I see the [optimization engine](https://tailwindcss.com/docs/optimizing-for-production) that removes unused CSS looks very promising.

Reply:
> Is SN planning on moving on to it? I see the [optimization engine](https://tailwindcss.com/docs/optimizing-for-production) that removes unused CSS looks very promising.

Maybe in the future. But for now the plan is to eventually move utility classes from this repo to [StyleKit](https://github.com/standardnotes/StyleKit) where part of our existing utility classes already are, so that editors can use them as well.